documentation/LoginAndLogoutFlow.md

```diff
@@ -75,8 +75,8 @@ The user login process is as follows:
    `flowManager.processMatch.priorities`. As with everything else in this flow, the sequence and ordering of the steps
    in the matchmaking process can be modified by the config/setup being used, and this document won't dive into the
    details of this flow except for some general observation:
-   1. Generally some prioritized steps: `preProcess`, `matchMakerDispatcher`, `runContextManager` and `transform` will
-      be run (in that order).
+   1. Generally some prioritized steps: `preProcess`, `matchMakerDispatcher`, `updsteActivePrefsSetName` and `transform`
```

Comment:
typo. updste -> update?

Reply:
and there's another one in `documentation/CloudBasedFlow.md`